WordPress/src/main/res/values/key_strings.xml

```diff
@@ -356,16 +356,16 @@
     </string-array>
 
     <string-array name="site_settings_image_quality_entries" translatable="false">
-        <item>@string/media_encoder_quality_80</item>
-        <item>@string/media_encoder_quality_85</item>
-        <item>@string/media_encoder_quality_90</item>
-        <item>@string/media_encoder_quality_95</item>
-        <item>@string/media_encoder_quality_100</item>
+        <item>@string/media_encoder_quality_low</item>
+        <item>@string/media_encoder_quality_medium</item>
+        <item>@string/media_encoder_quality_high</item>
+        <item>@string/media_encoder_quality_very_high</item>
+        <item>@string/media_encoder_quality_maximum</item>
     </string-array>
 
     <string-array name="site_settings_image_quality_values" translatable="false">
+        <item>70</item>
         <item>80</item>
-        <item>85</item>
         <item>90</item>
         <item>95</item>
```

Comment:
I'd like to note that the _Very High_ value wasn't added in the iOS version: https://github.com/wordpress-mobile/WordPress-iOS/pull/21981/files#diff-8dba2169bbf9087083db06c81198cd41a1b6a09b33f224fbed73f47dac842b15R22-R39.

Reply:
Seems we'd need to keep it as users on Android might have this preference set in previous versions.

Reply:
I think it'd be great to simplify this to match iOS, but agree it's trickier to do now that the setting has been out there for so long for Android users. I don't think we can really take it away.